src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -5,75 +5,77 @@
 import nextstep.users.domain.NsUser;
 
 import java.time.LocalDateTime;
+import java.util.List;
 
 public class Answer {
-    private Long id;
+	private Long id;
 
-    private NsUser writer;
+	private NsUser writer;
 
-    private Question question;
+	private Question question;
 
-    private String contents;
+	private String contents;
 
-    private boolean deleted = false;
+	private boolean deleted = false;
 
-    private LocalDateTime createdDate = LocalDateTime.now();
+	private LocalDateTime createdDate = LocalDateTime.now();
 
-    private LocalDateTime updatedDate;
+	private LocalDateTime updatedDate;
 
-    public Answer() {
-    }
+	public Answer() {
+	}
 
-    public Answer(NsUser writer, Question question, String contents) {
-        this(null, writer, question, contents);
-    }
+	public Answer(NsUser writer, Question question, String contents) {
+		this(null, writer, question, contents);
+	}
 
-    public Answer(Long id, NsUser writer, Question question, String contents) {
-        this.id = id;
-        if(writer == null) {
-            throw new UnAuthorizedException();
-        }
+	public Answer(Long id, NsUser writer, Question question, String contents) {
+		this.id = id;
+		if (writer == null) {
+			throw new UnAuthorizedException();
+		}
 
-        if(question == null) {
-            throw new NotFoundException();
-        }
+		if (question == null) {
+			throw new NotFoundException();
+		}
 
-        this.writer = writer;
-        this.question = question;
-        this.contents = contents;
-    }
+		this.writer = writer;
+		this.question = question;
+		this.contents = contents;
+	}
 
-    public Long getId() {
-        return id;
-    }
+	public Long getId() {
+		return id;
+	}
 
-    public Answer setDeleted(boolean deleted) {
-        this.deleted = deleted;
-        return this;
-    }
+	public Answer answerDeleted() {
```

Comment:
```suggestion
	public Answer answerDeleted() {
		if (!isOwners(writer)) {
			throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
		}
```
삭제할 권한이 있는지에 대한 체크를 이 메서드 내에서 하면 어떨까?